src/dashboards/hacs-dashboard.ts

```diff
@@ -85,16 +88,18 @@ const FILTER_SCHEMA = memoize(
         value: "",
       },
       {
-        name: "base",
+        name: "status",
         selector: {
           select: {
-            options: BASE_FILTER_OPTIONS.map((filter) => ({
-              value: filter,
-              label: localizeFunc(
-                // @ts-ignore
-                `common.${filter}`,
-              ),
-            })),
+            options: ["pending-restart", "pending-upgrade", "installed", "new", "default"].map(
```

Comment:
```suggestion
            options: Object.keys(STATUS_ORDER).map(
```
To ennsure that they are in sync.